lib/onc_certification_g10_test_kit/multi_patient_api.rb

```diff
@@ -9,9 +9,14 @@ class MultiPatientAPIGroup < Inferno::TestGroup
 
     input_instructions %(
       Register Inferno as a bulk data client with the following information, and
-      enter the client id and client registration in the appropriate fields. This
-      set of tests only checks the Group export. Enter the group export
+      enter the client id and client registration in the appropriate fields.
+      This set of tests only checks the Group export. Enter the group export
       information in the appropriate box.
+
+      If the server supports it, register Inferno with the following JWK Set
```

Comment:
```suggestion
      Register Inferno with the following JWK Set
```

Reply:
I did this in a separate commit to deal with line wrapping.